docs/README.md

```diff
@@ -10,33 +10,43 @@
+| [B2B Organizations](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-organizations) | Enables you to group B2B users into organizations, to which you can assign specific payment methods, price tables, and product collections. This allows all of the organization’s users to share the same commercial conditions. Each organization is further segmented into one or more cost centers, with its own shipping addresses which are available to that cost center's users during checkout. |
+| [B2B Checkout Settings](https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-checkout-settings) | Extends the checkout experience for users who are members of B2B organizations. With this app, you can allow specific payment methods for the user’s organization, provide pre-filled addresses based on each user’s cost center, include a purchase order number in the order, and allow customers to create an order quote before making an actual purchase. |
+| [B2B Orders History](https://github.com/vtex-apps/b2b-orders-history) | Provides a new **Orders** page under **My Account** so that B2B users can view all prior orders placed by users in their organization or cost center, depending on their role. |
 
-## B2B Orders History
+Make sure you read each app’s documentation to learn more about their capabilities and configurations.
 
-An app that provides a new order history page under My Account so that B2B users can view all prior orders placed by their organization or cost center (depending on role).
 
-⚠️ After installing the B2B Suite, make sure to hide the default Order History link within My Account by configuring `vtex.my-account`. This can be done at this URL: `https://{{accountName}}.myvtex.com/admin/apps/vtex.my-account@1.x/setup/`
+## Related apps
+
+For additional functionalities, you can also install the following apps, which automatically communicate with the ones in the B2B Suite.
+
+| **App name** | **Description** |
+|---|---|
+| [Storefront Permissions UI](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions-ui) | Provides an interface for the [Storefront Permissions](https://developers.vtex.com/vtex-developer-docs/docs/vtex-storefront-permissions) app, communicates with its stored roles and permissions and  provides the following features for this scenario: <ul><li><b>Roles management:</b> allows VTEX Admin users to manage B2B roles and associated app permissions through an interface.</li> <li><b>Theme blocks configuration:</b> enables conditional theme blocks so that only users with the required permissions can access determined parts of the content in your storefront.</li></ul> |
```

Comment:
The B2B Suite already depends on Storefront Permissions UI, so it's not necessary to list it here. Perhaps some of this information could be included in the earlier section about Storefront Permissions.